### PR TITLE
Fix, action confirmation disabling

### PR DESCRIPTION
**Pull request by mikitex70**
Fixed action disabling.

When using action with `confirmation=None`, a confirmation dialog still popups, with `None` message.
This is because the `None` Python value is translated in a `None` Javascript string, which is different from a `null` value and then `!!confirmation` is always `true`.

The fix checks for a `None` string value and convert it to a `false` Javascript value.

### CODE REVIEW

**Comment by mik-laj** [comment]
https://github.com/dpgaspar/Flask-AppBuilder/blob/ce6caa255a6b0e279d1e7dc8b2e51bc34cf60e94/flask_appbuilder/templates/appbuilder/general/lib.html#L78
```diff
				$('.{{action.name}}_menu_item').on('click', function(){
-				    return modelActions.execute_multiple('{{action.name}}','{{action.confirmation}}');
+                                   {% if confirmation %}
+				        return modelActions.execute_multiple('{{action.name}}','{{action.confirmation}}');
+                                   {% else %}
+				        return modelActions.execute_multiple('{{action.name}}', false);
+                                   {% endif %}
				})
```

**Comment by mikitex70** [comment]
> https://github.com/dpgaspar/Flask-AppBuilder/blob/ce6caa255a6b0e279d1e7dc8b2e51bc34cf60e94/flask_appbuilder/templates/appbuilder/general/lib.html#L78
> 
> ```diff
> 				$('.{{action.name}}_menu_item').on('click', function(){
> -				    return modelActions.execute_multiple('{{action.name}}','{{action.confirmation}}');
> +                                   {% if confirmation %}
> +				        return modelActions.execute_multiple('{{action.name}}','{{action.confirmation}}');
> +                                   {% else %}
> +				        return modelActions.execute_multiple('{{action.name}}', false);
> +                                   {% endif %}
> 				})
> ```

I don't understand: there was already a pull request for the issue, or this is a suggestion for a better way to fix it?

**Comment by mik-laj** [comment]
I think that this problem should be solved before it appears.  what do you think to fix it this way?

**Comment by mikitex70** [comment]
> I think that this problem should be solved before it appears. what do you think to fix it this way?

I like your solution, a lot better than mine.

**Comment by mik-laj** [comment]
Can you update your PR and include my solution?

File: flask_appbuilder/templates/appbuilder/general/lib.html [hunk 1]
```diff
@@ -75,7 +75,11 @@
 		{% for action_key in actions %}
         {% set action = actions.get(action_key) %}
 				$('.{{action.name}}_menu_item').on('click', function(){
-				    return modelActions.execute_multiple('{{action.name}}','{{action.confirmation}}');
+                    {% if confirmation %}
```
Comment (dpgaspar) [inline flask_appbuilder/templates/appbuilder/general/lib.html:78]: `confirmation` comes from where?

Comment (dpgaspar) [inline flask_appbuilder/templates/appbuilder/general/lib.html:78]: indentation looks really bad here, could you take the chance and fix it also?

Comment (mikitex70) [reply flask_appbuilder/templates/appbuilder/general/lib.html:78]: I don't know, I supposed suggestion from @mik-laj was correct.
Let me do some check.

Comment (mik-laj) [reply flask_appbuilder/templates/appbuilder/general/lib.html:78]: It should be action.confirmation

Comment (mikitex70) [reply flask_appbuilder/templates/appbuilder/general/lib.html:78]: In the file there was tabs and spaces before my editing. I've tried to keep indentation using tabs...
I will replace all tabs with spaces to keep indentation coherent with any editor.

Comment (dpgaspar) [reply flask_appbuilder/templates/appbuilder/general/lib.html:78]: Thank you!

**Pull request merged**